### DISCUSSION TITLE
[Feature]  Upload an image or several selected images

### DISCUSSION THREAD

**Discussion by WarlockFish** [immich-app/immich#3965] [Feature Request]
### The feature

now i find the immich app only upload the entire directory ，  What should I do when uploading a single file。
Can you add a feature to use Immichapp to open and upload photos when sharing them from the system's photo album?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2659
